webrender_api/src/api.rs

```diff
@@ -814,7 +887,10 @@ impl RenderApi {
                     flags: HitTestFlags)
                     -> HitTestResult {
         let (tx, rx) = channel::msg_channel().unwrap();
-        self.send(document_id, DocumentMsg::HitTest(pipeline_id, point, flags, tx));
```

Comment:
Is this line longer than 100 characters? If not, I think this rewrapping is unnecessary.